### PR TITLE
Editorial changes

### PR DESCRIPTION
**Pull request by kenchris**
Rename to Orientation Sensor and other editorial changes

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -70,21 +70,28 @@ spec:generic-sensor-1;
 Introduction {#intro}
 ============
 
-The DeviceOrientationSensor API extends the Generic Sensor API [[GENERIC-SENSOR]]
-to provide information that describes the device's physical orientation in a stationary
-three dimensional Cartesian coordinate system.
+The OrientationSensor API extends the Generic Sensor API [[GENERIC-SENSOR]]
+to provide information that describes the device's physical orientation.
+
+The base object describes the orientation in relation to the earths magnetic
```
Comment (alexshalamov) [inline index.bs]: The base object describes the rotation in relation to <a>Earth's reference coordinate system</a>?

"Earth's reference coordinate system" is already defined in model section, so we can ref it. Also "local coordinate system" is defined term that we can refer to.

Comment (kenchris) [inline index.bs]: OK, fixed. I hope this is what you wanted

File: index.bs [hunk 2]
```diff
@@ -100,10 +108,11 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
...
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The orientation sensors's associated {{Sensor}} subclass is the {{OrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer). The type of fusion
+is platform specific and can happen in software or hardware, ie. on a sensor hub.
```
Comment (pozdnyakov) [inline index.bs]: nit: i.e.

**Comment by kenchris** [comment]
Fixed

**Pull request merged**